exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/EndToEndTest.java

```diff
@@ -0,0 +1,134 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.cloud.trace.v2.TraceServiceClient;
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import com.google.devtools.cloudtrace.v2.ProjectName;
+import com.google.devtools.cloudtrace.v2.Span;
+import com.google.devtools.cloudtrace.v2.TruncatableString;
+import com.google.protobuf.BoolValue;
+import com.google.rpc.Status;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.TraceId;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.eq;
+
+@RunWith(JUnit4.class)
+public class EndToEndTest {
+
+  private static final String PROJECT_ID = "project-id";
+  private static final Map<String, AttributeValue> FIXED_ATTRIBUTES = new HashMap<>();
+  private static final TraceId TRACE_ID = new TraceId(321, 123);
+  private static final SpanId SPAN_ID = new SpanId(12345);
+  private static final SpanId PARENT_SPAN_ID = new SpanId(54321);
+  private static final String SPAN_NAME = "MySpanName";
+  private static final long START_EPOCH_NANOS = TimeUnit.SECONDS.toNanos(3000) + 200;
+  private static final long START_SECONDS = TimeUnit.NANOSECONDS.toSeconds(START_EPOCH_NANOS);
+  private static final int START_NANOS = (int) (START_EPOCH_NANOS - TimeUnit.SECONDS.toNanos(START_SECONDS));
+  private static final long END_EPOCH_NANOS = TimeUnit.SECONDS.toNanos(3001) + 255;
+  private static final long END_SECONDS = TimeUnit.NANOSECONDS.toSeconds(END_EPOCH_NANOS);
+  private static final int END_NANOS = (int) (END_EPOCH_NANOS - TimeUnit.SECONDS.toNanos(END_SECONDS));
+  private static final String SERVER_PREFIX = "Recv.";
+  private static final String AGENT_LABEL_KEY = "g.co/agent";
+  private static final String OPEN_TELEMETRY_LIBRARY_VERSION = "0.3.0";
```

Comment:
This will start failing after you rebase, because of the merged #14 https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/7fffee6b85c12bc3057c6112efda40a480038485/exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceTranslator.java#L30

Reply:
As suggested above, I think you should just make a string constant in `AGENT_LABEL_VALUE_STRING` below to test the final value.

Reply:
Done, inlined with `AGENT_LABEL_VALUE_STRING` and switched to 0.6.0